s3/s3i_test.go

```diff
@@ -152,6 +153,30 @@ func killBucket(b *s3.Bucket) {
 	panic(message)
 }
 
+func (s *ClientTests) TestSignedUrl(c *C) {
+
+	b := testBucket(s.s3)
+	err := b.PutBucket(s3.PublicRead)
+	c.Assert(err, IsNil)
+
+	err = b.Put("name", []byte("test for signed URLs."), "text/plain", s3.PublicRead)
+	c.Assert(err, IsNil)
+	defer b.Del("name")
+
+	url, err := b.SignedURL("name", 24*time.Hour)
```

Comment:
could we add an attempt to access "name" without a signed url and an assert that it fails?